edx-platform/pearson-pathways-theme/lms/templates/logout.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+{% load staticfiles theme_pipeline %}
```

Comment:
This not being a nested template, it needs to declare de doctype. In this case, HTML5, isn't it?

Reply:
yes, but I was asking for the second line. I wonder if we can get rid of it since we are not using anything related to themes here. Makes sense? @elvinquero

Reply:
Without theme_pipeline I can't use the javascript tag. https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/theming/templatetags/theme_pipeline.py

Reply:
And I need staticfiles when doing     <script type="text/javascript" src="{% static 'js/logout.js' %}"></script>